nvflare/private/fed/server/job_cmds.py

```diff
@@ -202,6 +211,60 @@ def clone_job(self, conn: Connection, args: List[str]):
             return
         conn.append_success("")
 
+    def authorize_ls_job(self, conn: Connection, args: List[str]):
+        if len(args) < 2:
+            conn.append_error("syntax error: missing job_id")
+            return False, None
+
+        if len(args) > 3:
+            conn.append_error("syntax error: too many arguments")
+            return False, None
+
+        return self.authorize_job(conn=conn, args=args[:2])
+
+    def ls_job(self, conn: Connection, args: List[str]):
+        job_id = conn.get_prop(self.JOB_ID)
+
+        if len(args) == 2:
+            conn.append_string("job\nworkspace\n\nSpecify the job or workspace dir to see detailed contents.")
+            return
+        else:
+            file = args[2]
```

Comment:
suggest enforce arg to 4
And have something data_source = args[3]
data_source need to be either "workspace" or "job" (or job_config?)

Example Usage:

`ls_job [job_id] workspace log.txt`
`ls_job [job_id] job_config meta.json`?